src/main/java/frc/robot/RobotContainer.java

```diff
@@ -170,14 +174,16 @@ public Command getAutonomousCommand() {
 
   private void configureBindings() {
 
-    shootButton.whileTrue(new ShootCommand());
+    shootButton.whileTrue(new ShootCommand(operator));
```

Comment:
This isn't being used anymore, right? If shootButton is the trigger, we should just need the default command set, which we have below. I think we can remove this line.

Reply:
removed